eZ/Bundle/PlatformInstallerBundle/src/Command/InstallPlatformCommand.php

```diff
@@ -0,0 +1,149 @@
+<?php
+/**
+ * This file is part of the eZ Publish Kernel package
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformInstallerBundle\Command;
+
+use Doctrine\DBAL\Exception\ConnectionException;
+use EzSystems\PlatformInstallerBundle\Installer\Installer;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class InstallPlatformCommand extends ContainerAwareCommand
+{
+    /** @var \Doctrine\DBAL\Connection */
+    private $db;
+
+    /** @var \Symfony\Component\Console\Output\OutputInterface */
+    private $output;
+
+    const EXIT_DATABASE_NOT_FOUND_ERROR = 3;
+    const EXIT_GENERAL_DATABASE_ERROR = 4;
+    const EXIT_PARAMETERS_NOT_FOUND = 5;
+    const EXIT_UNKNOWN_INSTALL_TYPE = 6;
+    const EXIT_MISSING_PERMISSIONS = 7;
+
+    protected function configure()
+    {
+        $this->setName( 'ezplatform:install' );
+
+        $this->addArgument( 'type', InputArgument::REQUIRED, "The type of install" );
```

Comment:
There is now no help text describing which options exists, or?
Besides we should default to clean or?